src/components/radio-button/radio-button.e2e.ts

```diff
@@ -194,6 +194,19 @@ describe("calcite-radio-button", () => {
 
     expect(await first.getProperty("checked")).toBe(true);
     expect(await second.getProperty("checked")).toBe(false);
+
+    //in addition to .click() check with .callMethod("click") as well
```

Comment:
Let's rephrase this to give context behind the different clicks. The code already lets us know we're using `click` and `.callMethod("click")`, but doesn't explain why.

Reply:
See if this solves it.